activiti-engine/src/main/java/org/activiti/engine/impl/persistence/entity/integration/IntegrationContextEntity.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.engine.impl.persistence.entity.integration;
+
+import java.util.Date;
+
+import org.activiti.engine.impl.persistence.entity.Entity;
+
+public interface IntegrationContextEntity extends Entity {
+
+    String getExecutionId();
+
+    void setExecutionId(String executionId);
+
+    String getCorrelationId();
+
+    void setCorrelationId(String correlationId);
+
+    String getProcessDefinitionId();
+
+    void setProcessDefinitionId(String processDefinitionId);
+
+    String getProcessInstanceId();
+
+    void setProcessInstanceId(String processInstanceId);
+
+    Date getCreatedDate();
+
+    void setCreatedDate(Date createdDate);
```

Comment:
@erdemedeiros I don't see the data being sent away in here. We need to make sure that looking at this entity we can retrigger the IntegrationRequestEvent

Reply:
The data is not directly represented in this class, they are hold the the execution. Here we only have the executionId that will allow you two retrieve this data. See the [second pull request for more information](https://github.com/Activiti/activiti-cloud-runtime-bundle-service/pull/1/files#diff-4d3ed153efda1fa500c39d3e8c25a63fR65).